app/views/facilities/index.html.haml

```diff
@@ -1,9 +1,7 @@
 - if SettingsHelper.feature_on?(:equipment_list)
   = render "shared/nav/equipment_list_nav"
 
-.alert.alert-info
-  = text(".welcome")
-  = render_view_hook("after_welcome")
+.alert.alert-info= html(".welcome")
```

Comment:
I'm not sure if it's important or not, but I believe that `html` will be wrapped in a `div` or a `p`. Can you double check and confirm no extra padding or anything gets added here from the change. If it's a problem, you should be able to use `html(".welcome", inline: true)`.

Also, we tend to not use the leading dot when using the text helper (rather than the rails helper `t`). Not a big deal, though.

Reply:
I tested this actually, using either `text` or `html` there's no visual difference between them.  I can clean up that `.` though.